Resources/Prototypes/Entities/Mobs/Customization/Markings/kemonomimi_ears.yml

```
@@ -3,7 +3,7 @@
   bodyPart: HeadTop
   markingCategory: HeadTop
   speciesRestriction: [Human]
-  sponsorOnly: true
+  sponsorOnly: true # LPP-Sponsors
```

yamllint

[warning] too few spaces before comment

(comments)